iroh/src/client/blobs.rs

```diff
@@ -1,5 +1,49 @@
 //! API for blobs management.
-
+//!
+//! The main entry point is the [`Client`].
+//!
+//! ## Interacting with the local blob store
+//!
+//! ### Importing data
+//!
+//! There are several ways to import data into the local blob store:
+//!
+//! - [`add_bytes`](Client::add_bytes)
+//!   imports in memory data.
+//! - [`add_stream`](Client::add_stream)
+//!   imports data from a stream of bytes.
+//! - [`add_reader`](Client::add_reader)
+//!   imports data from an [async reader](tokio::io::AsyncRead).
+//! - [`add_from_path`](Client::add_from_path)
+//!   imports data from a file.
+//!
+//! The last method imports data from a file on the local filesystem.
+//! This is the most efficient way to import large amounts of data.
+//!
+//! ### Exporting data
+//!
+//! There are several ways to export data from the local blob store:
+//!
+//! - [`read_to_bytes`](Client::read_to_bytes) reads data into memory.
+//! - [`read`](Client::read) creates a [reader](Reader) to read data from.
```

Comment:
```suggestion
//! - [`read`](Client::read) creates a [`Reader`] to read data from.
```
This would make more clear that you're linking directly to a type which I usually prefer.  That's just my personal preference though, no written rule.